src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    operate: operate,
+    add: add,
+    subtract: subtract,
+    multiply: multiply,
+    divide: divide,
+    reset: reset,
+    result: 0,
+  };
+
+  function operate(operation, number = 0) {
+    operation(number);
+
+    return this;
+  }
+
+  function add(number) {
+    calculator.result += number;
```

Comment:
Consider using 'this.result' instead of 'calculator.result' to make the code more maintainable and flexible.